apps/api/src/routers/admin.py

```diff
@@ -258,7 +258,6 @@ async def checkin(
 ) -> None:
     """Check in participant at IrvineHacks."""
     try:
-        # TODO: non-hackers
         await participant_manager.check_in_applicant(uid, associate)
```

Comment:
Chore: could you update the function name (and docstring) to be more generic, e.g. `check_in_participant`.

Reply:
I meant the name of the function in `participant_manager`